client/js/application.ts

```diff
@@ -126,3 +126,21 @@ for (let i = 0; i < inputsWithOther.length; i++) {
 		}
 	});
 }
+
+let wordCountInputs = document.querySelectorAll("[data-max-word-count], [data-min-word-count]") as NodeListOf<HTMLInputElement | HTMLTextAreaElement>;
+for (let i = 0; i < wordCountInputs.length; i++) {
+	wordCountInputs[i].addEventListener("input", e => {
+		let target = e.target as HTMLInputElement | HTMLTextAreaElement;
```

Comment:
seems like this can be `const`?

Reply:
Good call. I'm too used to Rust at the moment where everything is immutable by default and it also uses `let`